ci/scripts/acceptance-tests.sh

```diff
@@ -2,37 +2,38 @@
 
 set -euo pipefail
 
-readonly TOOLSMITH_ENV_INPUT="${TOOLSMITH_ENV_INPUT:?must be set}"
-readonly DEFAULT_ORG="${DEFAULT_ORG:?must be set}"
-readonly DEFAULT_SPACE="${DEFAULT_SPACE:?must be set}"
-readonly SKIP_SSL_VALIDATION="${SKIP_SSL_VALIDATION:?must be set}"
-readonly ONLY_SHOW_STANDARD_STREAMS_ON_TEST_FAILURE="${ONLY_SHOW_STANDARD_STREAMS_ON_TEST_FAILURE:-true}"
-readonly DEPLOYMENT_DIRECTORY="$(mktemp -d)"
-
-declare API_HOST
-readonly API_PORT=443
-readonly USERNAME="admin"
-declare PASSWORD
+readonly METADATA_FILE="${METADATA_FILE:?must be set}"
+
+declare CF_API_HOST
+readonly CF_USERNAME="admin"
+declare CF_PASSWORD
+
 readonly CLIENT_ID="admin"
 declare CLIENT_SECRET
 
+readonly DEFAULT_ORG="test"
+readonly DEFAULT_SPACE="development"
+
+DEPLOYMENT_DIRECTORY="$(mktemp -d)"
+readonly DEPLOYMENT_DIRECTORY
```

Comment:
TIL this is the expected way to assign and declare a `readonly`: https://www.shellcheck.net/wiki/SC2155

Reply:
Shellcheck ftw!